doc/source/yt4differences.rst

```diff
@@ -23,60 +23,67 @@ Updating to yt 4.0 from Old Versions (and going back)
 Converting Old Scripts to Work with yt 4.0
 ------------------------------------------
 
-Energy and Momentum Field Name Conventions
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Fields representing energy and momentum quantities are now given names which
-reflect their dimensionality. For example, the ``"gas", "kinetic_energy"``
-field was actually a field for kinetic energy density, and so it has been
-renamed to ``"gas", "kinetic_energy_density"``. The old name still exists
-as an alias as of yt v4.0.0, but it will be removed in yt v4.1.0. See
-:ref:`deprecated_field_names` below for more information.
-
-Other examples include ``"gas", "specific_thermal_energy"`` for thermal
-energy per unit mass, and ``"gas", "momentum_density_x"`` for the x-axis
-component of momentum density. See :ref:`efields` for more information.
-
-Deprecated Field Names
-^^^^^^^^^^^^^^^^^^^^^^
-Certain field names are deprecated within yt v4.0.0, and will be removed in
-yt v4.1.0. For example, ``"gas", "kinetic_energy"`` has been renamed to
-``"gas", "kinetic_energy_density"``, though the former name has been added
-as an alias. Other fields, such as ``"gas", "cylindrical_tangential_velocity_absolute"``,
-are being removed entirely. When the deprecated field names are used for the
-first time in a session, a warning will be logged, so it is advisable to set
-your logging level to ``WARNING`` at a minimum to catch these.
-
-Colormaps
-^^^^^^^^^
-yt used to automatically register colormaps from ``cmocean``, unprefixed.
-This unfortunately became unsustainable with the 3.4 release of Matplotlib,
-in which colormaps with colliding names raise errors. The fix is to explicitly
-import the ``cmocean`` module and prefix ``cmocean`` colormaps (like
-``balance``) with ``cmo.``.  Note that this solution works with any
-yt-supported version of Matplotlib, but is not backward compatible with earlier
-versions of yt. Example:
-
-Old Script (will not work in the near future):
-
-.. code-block:: python
-
-    import yt
-
-    ds = yt.testing.fake_random_ds(1)
-    p = yt.SlicePlot(ds, "z", ("gas", "density"))
-    p.set_cmap(("gas", "density"), "balance")
-
-New Script:
-
-.. code-block:: python
-
-    import yt
-    import cmocean
-
-    ds = yt.testing.fake_random_ds(1)
-    p = yt.SlicePlot(ds, "z", ("gas", "density"))
-    p.set_cmap(("gas", "density"), "cmo.balance")
+After installing yt-4.0, you’ll want to change your old scripts in a few key
+ways. After accounting for the changes described in the list below, try
+running your script. If it still fails, the callback failures in python
+are fairly descriptive and it may be possible to deduce what remaining
```

Comment:
I wouldn't *guarantee* their clarity. Note that the native error messages in Python 3.10 received a nice polishing, though even with Python 3.10, getting an error message from >=3 levels bellow the function you're using is usually not that helpful, and a common occurence in my experience of the ecosystem, yt in particular.
```suggestion
running your script. If it still fails, the Python tracebacks
should be fairly descriptive and it may be possible to deduce what remaining
```